### DISCUSSION TITLE
VSC Extension Failing

### DISCUSSION THREAD

**Discussion by Quantalabs** [community/community#25004] [Codespaces]
<p>When using the Codespaces extension on VSC, every time I create a new codespace, it fails with the error:</p>
<p><code>Create New Codespace: List instances request failed with status 404. Correlation ID: E682:735A:37DE7C:44476D:60025F0A</code></p>
<p>Is this a bug, or is codespaces not setup properly?</p>

**Comment by jkeech** [comment]
<p>Hi <a class="mention" href="/quantalabs">@Quantalabs</a>! Sorry you are having issues.</p>
<p>Just to confirm, do you have access to the Codespaces beta? You can check by seeing if you have a Codespaces tab on the top of <a href="http://github.com">github.com</a>. If you are in the beta, then this error during creation is not expected. Are you able to create from the web instead of the VS Code extension?</p>

**Comment by jkeech** [comment, marked as answer]
<p>I confirmed that this is currently what happens if you sign in and manually invoke the Create New Codespace from the command palette and are not currently in the beta. I’ll work on updating the extension to provide a better user experience in this case, but unfortunately you’ll need to wait for beta access before you can use Codespaces. You can sign up at <a href="https://github.com/features/codespaces" class="inline-onebox">Codespaces · GitHub</a> if you haven’t done so already.</p>

**Comment by Quantalabs** [comment]
<p>Yes, this must be the problem. Is there a way to improve the error message?</p>

**Comment by jkeech** [comment]
<p><a class="mention" href="/quantalabs">@Quantalabs</a>, we’ve fixed this in version 0.6.0 to disable commands including creation for users who are not in the beta: <a href="https://marketplace.visualstudio.com/items/GitHub.codespaces/changelog" class="inline-onebox">Changelog | Visual Studio Marketplace</a>.</p>
<p>Version 0.6.0 requires VS Code 1.53+, which is in Insiders right now. Once VS Code stable gets updated to 1.53, version 0.6.0 should be installed in VS Code stable as well.</p>
